### DISCUSSION TITLE
There is some risk when using `nnictl update concurrency`

### DISCUSSION THREAD

**Discussion by xuehui1991** [microsoft/nni#3782] [Ideas]
**Short summary about the issue/question**:

In the evolution algorithm, the `concurrency` related to the `population_size`. If directly change the `concurrency`, may cause the experiment to fail or hurt the algorithm performance.

**Brief what process you are following**: 

**How to reproduce it**: 

**nni Environment**:
- nni version:
- nni mode(local|pai|remote):
- OS:
- python version:
- is conda or virtualenv used?: 
- is running in docker?:

**need to update document(yes/no)**:

**Anything else we need to know**:

**Comment by scarlett2018** [comment]
@xuehui1991 - is this problem only to evolution algorithm? 

shall we disable the concurrency changes for evolution algorithm or just through warnings to user about this but keep our original generic behavior?

**Comment by xuehui1991** [comment]
@scarlett2018  I don't think it only for evolution. I think we could add some tips for the button that we change the concurrency. I have discussed the UI design with OU YANG.

**Comment by scarlett2018** [comment]
> @scarlett2018 I don't think it only for evolution. I think we could add some tips for the button that we change the concurrency. I have discussed the UI design with OU YANG.

Thanks for the confirm, let's discuss on our Friday planning meeting.
